### ISSUE TITLE
Fix comments broken in #1876

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#1908]
#1876 updated proto file comments incorrectly, e.g.:

``// Maps from: data/src/main/java/org/oppia/data/backends/gae/model/GaeVoiceover.kt`` in voiceover.proto was incorrectly replaced with ``// Maps from: data/src/main/java/org.oppia.android.android/data/backends/gae/model/GaeVoiceover.kt``. It should be: ``// Maps from: data/src/main/java/org/oppia/android/android/data/backends/gae/model/GaeVoiceover.kt``.

This issue is tracking checking all proto files for these incorrect paths & fixing them.

**Comment by soamOne** [comment]
Hey @BenHenning @rt4914 Can I work on this issue?

**Comment by Victor-Titan** [comment]
I'd like to work on this issue

**Comment by BenHenning** [comment]
@soamOne it looks like @Victor-Titan already started on the issue. Feel free to pick another starter issue that's not yet claimed. :)

**Comment by soamOne** [comment]
Yes got it @BenHenning :)
